docs/troubleshooting.asciidoc

```diff
@@ -18,6 +18,17 @@ IMPORTANT: If you do so, *please attach your debug logs* so that we can analyze
 Upload the *complete* logs to a service like https://gist.github.com.
 The logs should include everything from the application startup up until the first request has been executed.
 
+[float]
+[[trouble-shooting-use-latest-agent]]
+=== Updating to latest agent version
+
+Agent is updated frequently and releases are not strongly tied to other components in the stack.
+
+As a result, trying to update to the most recently released agent version is often the recommended first troubleshooting
+step. Also, between releases using the https://github.com/elastic/apm-agent-java/blob/master/README.md#snapshots[latest-snapshot] might also be relevant.
```

Comment:
```suggestion
step. If possible, trying the https://github.com/elastic/apm-agent-java/blob/master/README.md#snapshots[latest-snapshot] is even preferable as it would include fixes that are not yet released.
```